File: OnlinerMoto/OnlinerMotoLoadMoreCell.m
//
//  OnlinerMotoLoadMoreCell.m
//  OnlinerMoto
//
//  Created by Igor Karpov on 24.11.2013.
//  Copyright (c) 2013 KarpovIV. All rights reserved.
//

#import "OnlinerMotoLoadMoreCell.h"

@implementation OnlinerMotoLoadMoreCell

- (void)setInitialState
{
    [self.activityIndicator stopAnimating];
    self.label.text = @"Load more";
}

- (void)setLoadingState
{
    self.label.text = @"Loading...";
    [self.activityIndicator startAnimating];
}

@end

File: OnlinerMoto/OnlinerMotoLoadMoreCell.swift
//
//  OnlinerMotoLoadMoreCell.swift
//  OnlinerMoto
//
//  Created by Igor Karpov on 13.11.2014.
//  Copyright (c) 2014 KarpovIV. All rights reserved.
//

import Foundation

class OnlinerMotoLoadMoreCell : UITableViewCell
{
    @IBOutlet weak var label: UILabel!
    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
    
    required init(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
    }
    
    func setInitialState()
    {
        self.activityIndicator.stopAnimating()
        self.label.text = "Load more"
    }
    
    func setLoadingState()
    {
        self.label.text = "Loading ..."
        self.activityIndicator.startAnimating()
    }
}